cfme/infrastructure/virtual_machines.py

```diff
@@ -512,6 +513,24 @@ def is_displayed(self):
         return self.in_infra_vms and self.title.text == expected_title
 
 
+class InfraVmCompareToolbar(View):
+    """The toolbar on the compare page"""
+    all = Button(title='All Attributes')
+    different = Button(title='Attributes with different values')
+    same = Button(title='Attributes with same values')
```

Comment:
Since below you are using the `toolbar` of the [`CompareView`](https://github.com/ManageIQ/integration_tests/blob/262c90289658390a4f31870dd5d5a29a44e8d087/cfme/common/__init__.py#L376), I'm don't think this class is necessary.